data/siggraph-2019/10.1145-3306346.3322952.json

```diff
@@ -0,0 +1,96 @@
+[
+    {
+        "Variant name": "main",
+        "Reviewer name": "Julie Digne <julie.digne@liris.cnrs.fr>",
+
+        "Is master variant (boolean)": true,
+        
+        "Is variant deprecated (boolean)": false,
+
+        "Operating system for the test": "Ubuntu 20.04",
+        
+        "Title": "RedMax: efficient & flexible approach for articulated dynamics",
+
+        "DOI": "10.1145/3306346.3322952",
+
+        "Year": 2019,
+
+        "ACM Keywords": [
+            "Physical simulation"
+        ],
+
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Animation and Simulation",
+
+        "Co-authors from academia (boolean)": true,
+
+        "Co-authors from industry (boolean)": true,
+
+        "ACM Open Access (boolean)": false,
+
+        "PDF on the authors' webpage / institution (boolean)": true,
+
+        "PDF on Arxiv or any openarchive initiatives (boolean)": false,
+        "Arxiv/OAI page URL": "",
+
+        "PDF URL": "http://faculty.cs.tamu.edu/sueda/projects/redmax/WWBHKS2019.pdf",
+        
+        "Project URL": "http://faculty.cs.tamu.edu/sueda/projects/redmax/",
+
+        "Code available (boolean)": true,
+        "Code URL": "https://github.com/sueda/redmax",
+        "Code URL2": "",
+        "MD5 sum (for archives)": "",
+        "MD5 sum (for archives) URL2": "",
+        "git/hg/svn commit hash or revision number": "299d70d9713385661ce94ee5f2011470002b46bf",
+        "git/hg/svn commit hash or revision number URL2": "",
+
+        "Software Heritage permalink": "",
```

Comment:
There is a SWH link for this one https://archive.softwareheritage.org/browse/origin/log/?origin_url=https://github.com/sueda/redmax

But I cannot cut/paste it from the phone..

Reply:
done